setup/kubernetes/k3s.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "K3s"
+description: Set up K3s on an Ubuntu machine to deploy Coder.
+---
+
+This article will show you how to install K3s onto a new Ubuntu 20.04 LTS
+machine for use with Coder.
+
+[K3s](https://k3s.io/) is a lightweight Kubernetes distribution that works well
+for single-node or multi-node clusters. This guide covers the installation
+of K3s onto a new Ubuntu 20.04 LTS machine, but if you want to install Coder on
```

Comment:
> This doc is currently Ubuntu-specific, but for no particular reason other than "it works." Should I test on some other operating systems or better describe requirements? The goal by mentioning the OS is that this works best on a new machine.

@bpmct I think this is fine, as long as we are explicit about what the guide covers, then we set the expectation with readers that we've tested with a fresh installation of Ubuntu 20.04. I don't think anyone can expect that we test on every possible OS and configuration, and stating the version we used to test here makes clear that this is the version someone would have to use in order to have the highest probability of success with this guide.

Reply:
Great, thanks :)